docs/stack/insight/_index.md

```diff
@@ -53,13 +57,51 @@ The CLI is accessible at any time within the application.
 Advanced command line interface with intelligent command auto-complete and complex data visualizations.
 * Built-in guides: you can conveniently discover Redis and Redis Stack capabilities using the built-in guides.
 * Command auto-complete support for all capabilities in Redis and Redis Stack.
-* Visualizations of your [RediSearch](https://oss.redis.com/redisearch/) index, queries, and aggregations
-* Visualizations of your [RedisGraph](https://oss.redis.com/redisgraph/) and [RedisTimeSeries](https://oss.redis.com/redistimeseries/) data.
+* Visualizations of your [RediSearch](https://oss.redis.com/redisearch/) index, queries, and aggregations.
+* Visualizations of your [RedisGraph](https://oss.redis.com/redisgraph/).
+
+  <img src="/docs/stack/insight/images/Workbench_Graph.png">
+
+* Visualizations of your [RedisTimeSeries](https://oss.redis.com/redistimeseries/) data.
+
+  <img src="/docs/stack/insight/images/Workbench_TimeSeries.png">
+
+## Tools
+
+### Database analysis
+
+Use the database analysis tool to optimize the performance and memory usage of your Redis database. Check data type distribution and memory allocation and review the summary of key expiration time and memory to be freed over time. Inspect the top keys and namespaces sorted by consumed memory or key length and count of keys, respectively. Capture and track the changes in your database by viewing historical analysis reports. Next figure shows a sample database analysis report.
+
+<img src="/docs/stack/insight/images/database_analysis.png">
+
+### Redis Streams support
+
+Create and manage streams by adding, removing, and filtering entries per timestamp. To see and work with new entries, enable and customize the automatic refresh rate.
+
+View and manage the list of consumer groups. See existing consumers in a given consumer name as well as the last messages delivered to them. Inspect the list of pending messages, explicitly acknowledge the processed items, or claim unprocessed messages via RedisInsight.
+
+<img src="/docs/stack/insight/images/streams.png">
+
+### Search capabilities
+
+If you're using Redis or Redis Stack's indexing, querying, or full-text search capabilities, RedisInsight provides UI controls to quickly and conveniently run search queries against a preselected index. You can also create a secondary index of your data in a dedicated pane.
```

Comment:
this is not correct.  It's only for Redis stacks' indexing capabilities. need to go back to check the blog now

Reply:
updated!